src/core/bullet.js

```diff
@@ -0,0 +1,57 @@
+import { Sprite, Assets } from "pixi.js";
+
+export class bulletProjectile {
+  constructor(bulletX, bulletY, app) {
+    this.bulletX = bulletX;
+    this.bulletY = bulletY;
+    this.app = app;
+
+    // when sliders are implemented, these values should be manipulated roughly 
+    this.bulletSpeedX = 100;
+    this.bulletSpeedY = 50;
+    this.gravity = 5;
+  }
+
+  getX() {
+    return this.bulletX;
+  }
+
+  getY() {
+    return this.bulletY;
+  }
+
+  async initialiseSprite() {
+    // load texture of player, and convert into sprite.
+    const texture = await Assets.load('assets/images/bullet.png'); // 'await' keyword used for asynchronous texture loading
+    const sprite = Sprite.from(texture);
```

Comment:
More specific name

Reply:
Should it be a const if we keep changing it?
I get the image being a const but the sprite created from it too? Not sure it's the best approach but open to being wrong